From 332e14f74e5580af746411975c70c555a0737109 Mon Sep 17 00:00:00 2001
From: Andreas Baldeau <andreas@baldeau.net>
Date: Sun, 4 Nov 2018 00:19:57 +0100
Subject: [PATCH] Disable menu entries in recovery

* Wipe data/factory reset
* Mount /system
* View recovery logs
* Run graphics test

---
 bootable/recovery/device.cpp | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/device.cpp b/device.cpp
index f881daf..b90520c 100644
--- a/bootable/recovery/device.cpp
+++ b/bootable/recovery/device.cpp
@@ -21,13 +21,13 @@ static const char* MENU_ITEMS[] = {
   "Reboot to bootloader",
   "Apply update from ADB",
   "Apply update from SD card",
-  "Wipe data/factory reset",
 #ifndef AB_OTA_UPDATER
+  "Wipe data/factory reset",
   "Wipe cache partition",
-#endif  // !AB_OTA_UPDATER
   "Mount /system",
   "View recovery logs",
   "Run graphics test",
+#endif  // !AB_OTA_UPDATER
   "Run locale test",
   "Power off",
   nullptr,
@@ -38,13 +38,13 @@ static const Device::BuiltinAction MENU_ACTIONS[] = {
   Device::REBOOT_BOOTLOADER,
   Device::APPLY_ADB_SIDELOAD,
   Device::APPLY_SDCARD,
-  Device::WIPE_DATA,
 #ifndef AB_OTA_UPDATER
+  Device::WIPE_DATA,
   Device::WIPE_CACHE,
-#endif  // !AB_OTA_UPDATER
   Device::MOUNT_SYSTEM,
   Device::VIEW_RECOVERY_LOGS,
   Device::RUN_GRAPHICS_TEST,
+#endif  // !AB_OTA_UPDATER
   Device::RUN_LOCALE_TEST,
   Device::SHUTDOWN,
 };
-- 
2.19.1

